pkgs/development/python-modules/objax/default.nix

```diff
@@ -40,6 +40,10 @@ buildPythonPackage rec {
     "objax"
   ];
 
+  # Tests require tensorflow which causes conflicts in the closure due to duplicates of the
+  # following packages: grpcio, protobuf, tensorboard and tensorboard-plugin-profile.
+  doCheck = false;
```

Comment:
Have you seen the approach in https://github.com/NixOS/nixpkgs/blob/3fdb70efbba3047c2ed2cceacf3d63ae4e5bdf5f/pkgs/development/python-modules/cleanlab/default.nix#L47-L52 (https://github.com/NixOS/nixpkgs/pull/263048)? Specifically, the disabling of `catchConflicts` for the `doCheck` passthru test.

Also is it specifically protobuf that causes conflicts?

Reply:
You can also just make passthru.tests.check and ``dontInstall``.
Since the purpose is for testing, and it's  a passthru attribute, you can work around the conflict without affecting the actual used package.

Reply:
Unfortunately, what I just suggested is not applicable here.
It was something that was only working because the conflict was direct and not indirect like this is.

Reply:
In this case, there is no direct need for the `passthru` trick (no circular dependency problem). Hence, I have just set `catchConflicts` to `false` in the main scope. Is that a problem ?